OpenRA.Mods.Common/UpdateRules/Rules/20200202/RenameWithNukeLaunch.cs

```diff
@@ -0,0 +1,35 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	class RenameWithNukeLaunch : UpdateRule
+	{
+		public override string Name { get { return "Renamed WithNukeLaunchAnimation and Overlay"; } }
+		public override string Description
+		{
+			get
+			{
+				return "The traits have been generalized to apply for all support powers.";
```

Comment:
The description is supposed to contain enough information to (at least try to) document the changes that a modder would need to do manually if they don't run the rule. Please say explicitly that `WithNukeLaunchAnimation` has been renamed to `WithSupportPowerActivationAnimation` and `WithNukeLaunchOverlay` to `WithSupportPowerActivationOverlay`.

Reply:
Rephrased it and moved it to the other bleed rules.